### PR TITLE
Reload file push.php

### PR DESCRIPTION
**Pull request by Edsol**
there was a problem with loading the file using the cakephp scheduler:

`bin/cake scheduler run`

returns an error:

`Exception: No API key set.`

I've edit constructor to load file

### CODE REVIEW

File: src/Adapter/Fcm.php [hunk 1]
```diff
@@ -9,345 +9,353 @@
...
+	];
+
+	/**
+	 * FcmAdapter constructor.
+	 *
+	 * @throws \Exception
+	 */
+	public function __construct()
+	{
+		if (Configure::check('Push.adapters.Fcm') == false) {
```
Comment (ker0x) [inline src/Adapter/Fcm.php]:
```suggestion
		if (Configure::check('Push.adapters.Fcm') === false) {
```

File: src/Adapter/Fcm.php [hunk 2]
```diff
@@ -9,345 +9,353 @@
...
+			try {
+				Configure::load('push');
+			} catch (\Exception $e) {
+				exit($e->getMessage() . "\n");
```
Comment (ker0x) [inline src/Adapter/Fcm.php]:
```suggestion
				throw new \RuntimeException($e->getMessage());
```

**Review by ker0x — COMMENTED** [review]
Hello @Edsol an thank you for your PR.

I made some feedback, if you can take a look! Also can you please kepp indentation with 4 spaces instead of tabs, it will be much appreciate 🙏!

**Comment by Edsol** [comment]
> Hello @Edsol an thank you for your PR.
> 
> I made some feedback, if you can take a look! Also can you please kepp indentation with 4 spaces instead of tabs, it will be much appreciate pray!

done :wink:

**Comment by Edsol** [comment]
@ker0x please, can you merge it? Thank you

**Pull request merged**